S3/ConnMan.py

```diff
@@ -73,9 +86,20 @@ def _ssl_context():
         cafile = cfg.ca_certs_file
         if cafile == "":
             cafile = None
+        certfile = cfg.ssl_cert_file
+        if certfile == "":
+            certfile = None
+        keyfile = cfg.ssl_key_file
+        if keyfile == "":
+            keyfile = None
+
```

Comment:
For the previous blocks, including cafile, you can simplify it:
ex.:
> cafile = cfg.ca_certs_file or None

(There is a lot of non really clean code all other s3cmd, but it is legacy, and not a code style to follow :-) )